dependencyIndexingSchedulerHandler panic on sourcegraph.com

https://console.cloud.google.com/errors/COv8nuSbjYm8UQ?time=P30D&project=sourcegraph-dev

```
panic: runtime error: index out of range [1] with length 1

goroutine 33773551 [running]:
github.com/sourcegraph/sourcegraph/enterprise/cmd/frontend/internal/codeintel/background/indexing.(*dependencyIndexingSchedulerHandler).Handle(0xc00172adc0, 0x68b1728, 0xc0676f7d40, 0x68d4310, 0xc00dc88820, 0x6856520, 0xc0489eca50, 0x0, 0x0)
	github.com/sourcegraph/sourcegraph/enterprise/cmd/frontend/internal/codeintel/background/indexing/dependency_indexing_scheduler.go:96 +0x725
github.com/sourcegraph/sourcegraph/internal/workerutil/dbworker.(*handlerShim).Handle(0xc0020fbde0, 0x68b1728, 0xc0676f7d40, 0x68c88f8, 0xc048cd26f0, 0x6856520, 0xc0489eca50, 0x0, 0x0)
	github.com/sourcegraph/sourcegraph/internal/workerutil/dbworker/handler_shim.go:25 +0x8f
github.com/sourcegraph/sourcegraph/internal/workerutil.(*Worker).handle(0xc001d4eb40, 0x68c88f8, 0xc048cd26f0, 0x6856520, 0xc0489eca50, 0x6473738, 0xc05fa6afa8)
	github.com/sourcegraph/sourcegraph/internal/workerutil/worker.go:201 +0x207
github.com/sourcegraph/sourcegraph/internal/workerutil.(*Worker).dequeueAndHandle.func2(0xc001d4eb40, 0x6856520, 0xc0489eca50, 0x68c88f8, 0xc048cd26f0)
	github.com/sourcegraph/sourcegraph/internal/workerutil/worker.go:179 +0xb8
```

--- a/enterprise/cmd/frontend/internal/codeintel/background/indexing/dependency_indexing_scheduler.go
+++ b/enterprise/cmd/frontend/internal/codeintel/background/indexing/dependency_indexing_scheduler.go
@@ -93,7 +93,7 @@ func (h *dependencyIndexingSchedulerHandler) Handle(ctx context.Context, tx dbwo
 		return nil
 	}
 	if len(errs) == 1 {
-		return errs[1]
+		return errs[0]
 	}
 
 	return multierror.Append(nil, errs...)

